x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -196,30 +310,34 @@ elastic-agent run [global-flags]
 
 These flags are valid whenever you run `elastic-agent` on the command line.
 
-`-c <string>`::
+`--c <string>`::
 The policy file to use. If not specified, {agent} uses
-`{path.home}/elastic-agent.yml`.
+`{path.config}/elastic-agent.yml`.
+
+//REVIEWERS: Previously we decided that we would not document the debug selector
+//option (--d). Is that still true?
```

Comment:
I don't know about that decision but I would not add it.

Reply:
we can always add it later so leaving out for now.